src/pages/extending-agent-app/index.mdx

```diff
@@ -8,7 +8,7 @@ desc: "You can extend the LiveChat Agent App with widgets, buttons, and other UI
 
 # Introduction
 
-This document presents how you can change and extend the LiveChat App and its interface, so it is personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
+This document presents how you can change and extend the LiveChat and HelpDesk Apps and their interfaces, so they're personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
```

Comment:
```suggestion
This document presents how you can change and extend the LiveChat and HelpDesk products and their interfaces, so they're personalized and follow your needs. You can do it by configuring your app to be displayed in one of the available **App locations** described below.
```

Reply:
Let's do it this way so there's a difference between our products and apps built by devs